.travis.yml

```diff
@@ -97,10 +97,12 @@ jobs:
     install:
     - cargo clippy --version | grep $CLIPPY_VERS || cargo install clippy --force --vers $CLIPPY_VERS
     script:
-    - cargo clippy --all
+    - cargo clippy --all -- -D warnings
 
   # Tests
   - env: FEATURE=test
+    install:
+    - export RUSTFLAGS="$RUSTFLAGS -D warnings"
```

Comment:
Does it make more sense to apply `-D warnings` in this way (on the per-job basis), or would it make more sense to redefine the global setting of the `RUSTFLAGS` variable? The first way is more flexible, but can be easier to forget: e.g., `-D warnings` is not active for benchmarking job (not sure it should).

Reply:
I'd say let's define `RUSTFLAGS` once globally, and, if/when we find a need to disable it for a particular job, switch to per-job setting.

Reply:
> if/when we find a need to disable it for a particular job, switch to per-job setting.

I would even prefer to disable such warnings locally with attributes.